$NetBSD: patch-Makefile,v 1.1 2018/09/19 09:33:52 leot Exp $

Properly set permissions of header files (do not need executable bit).

--- Makefile.orig	2016-08-05 17:37:12.000000000 +0000
+++ Makefile
@@ -98,7 +98,7 @@ installlib: libxdo.$(LIBSUFFIX)
 .PHONY: installheader
 installheader: xdo.h
 	install -d $(DINSTALLINCLUDE)
-	install xdo.h $(DINSTALLINCLUDE)/xdo.h
+	install -m 644 xdo.h $(DINSTALLINCLUDE)/xdo.h
 
 .PHONY: installman
 installman: xdotool.1
